packages/dashboard-frontend/tests/services/remote-cached-data-provider.test.js

```diff
@@ -0,0 +1,78 @@
+import { describe, expect, it } from "@jest/globals";
+import { waitFor } from "@testing-library/react";
+import { fetchJson } from "../../src/fetch/fetch-json";
+import { getItem, setItem } from "../../src/services/cache";
+import { RemoteCachedDataProvider } from "../../src/services/remote-cached-data-provider";
+
+jest.mock( "../../src/services/cache", () => ( {
+	getItem: jest.fn(),
+	setItem: jest.fn(),
+} ) );
+jest.mock( "../../src/fetch/fetch-json" );
+
+describe( "RemoteCachedDataProvider", () => {
+	beforeAll( () => {
+		fetchJson.mockImplementation( () => {
```

Comment:
Any reason you went with this style vs the one you went for with the cache get/set?

```js
jest.mock( "../../src/fetch/fetch-json", () => ( {
	fetchJson: jest.fn().mockImplementation( () => Promise.resolve( "result" ) ),
} ) );
```

Reply:
Nope, made it the same style [here](https://github.com/Yoast/wordpress-seo/pull/22216/commits/18a4890fe7eb74298f1c8e9fe13922564a4d3103).